src/components/Comments/Comment.js

```diff
@@ -147,7 +150,8 @@ const Comment = ({
         const members = [comment?.user?.id, profile.myProfile.user_id];
         await createSignChat(members, selectedUser);
       } else {
-        await sendMessageDM(comment?.id, 'comment', 'SIGNED');
+        console.log('feedId', feedId);
+        await sendMessageDM(comment?.id, 'comment', 'SIGNED', feedId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendMessageDM` function now includes an additional parameter `feedId`. Ensure that all calls to this function have been updated to include this new argument.



<!-- This is an auto-generated comment by OSS CodeRabbit -->